addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/MongoDbInjectionScanRule.java

```diff
@@ -55,83 +59,46 @@ public class MongoDbInjectionScanRule extends AbstractAppParamPlugin {
     // Prefix for internationalised messages used by this rule
     private static final String MESSAGE_PREFIX = "ascanalpha.mongodb.";
     // Constants
+    private static final String RULE_SLEEP_TIME = RuleConfigParam.RULE_COMMON_SLEEP_TIME;
     private static final String ALL_DATA_ATTACK = "alldata";
     private static final String CRASH_ATTACK = "crash";
     private static final String SLEEP_ATTACK = "sleep";
     private static final String JSON_ATTACK = "json";
     private static final String AUTH_BYPASS_ATTACK = "authbypass";
     private static final String JSON_TOKEN = "$ZAP";
-    private int SLEEP_SHORT_TIME, SLEEP_LONG_TIME;
-    private static final String INSERT_SHORT_TIME = "INSERT_SHORT_TIME",
-            INSERT_LONG_TIME = "INSERT_LONG_TIME";
-    private static final int SHORT_THRESHOLD = 1500;
-    private static final int LONG_THRESHOLD = 3000;
+    private static final int BLIND_REQUEST_LIMIT = 5;
+    private static final int DEFAULT_TIME_SLEEP_SEC = 10;
```

Comment:
Do these need to be different from the other scan rule? (The name of the first constant could also exactly match the other.)